sdk/js/src/sui/template/coin.move

```diff
@@ -0,0 +1,28 @@
+module wrapped_coin::coin {
+    use sui::transfer::{Self};
+    use sui::tx_context::{Self, TxContext};
+
+    use token_bridge::asset_meta::{Self, AssetMeta};
+    use token_bridge::create_wrapped::{Self};
+
+    struct COIN has drop {}
+
+    const VAA: vector<u8> = x"{{VAA_BYTES}}";
+
+    fun init(witness: COIN, ctx: &mut TxContext) {
+        transfer::public_transfer(
+            create_wrapped::prepare_registration(
+                witness,
+                VAA,
+                ctx
+            ),
+            tx_context::sender(ctx)
+        );
+    }
+
+    public fun token_meta(): AssetMeta {
```

Comment:
We may not want this here unless it'll be used to test something (which I didn't find in this PR).

Reply:
Removed